### PR TITLE
Create static lessons page

### PR DESCRIPTION
**Pull request by ayomanetsan**

### CODE REVIEW

File: frontend/src/app/modules/lessons/lessons-page/lessons-page.component.sass [hunk 1]
```diff
@@ -0,0 +1,302 @@
+@import '../../../../styles/colors'
+@import url('https://fonts.googleapis.com/css2?family=Barlow:wght@400;500;700&display=swap')
+
+*
+    margin: 0
```
Comment (Anatolii-Ivanov) [inline frontend/src/app/modules/lessons/lessons-page/lessons-page.component.sass:5]: Please no global styles in general files

File: frontend/src/app/modules/lessons/lessons-page/lessons-page.component.sass [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+
+.container
+    width: 75%
+    background: $gray-100
+    margin: auto
+    padding: 31px 57px
```
Comment (Anatolii-Ivanov) [inline frontend/src/app/modules/lessons/lessons-page/lessons-page.component.sass:11]: round such values to 30/55

Comment (Anatolii-Ivanov) [reply frontend/src/app/modules/lessons/lessons-page/lessons-page.component.sass:11]: Everywhere

File: frontend/src/app/modules/lessons/lessons-page/lessons-page.component.sass [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+.lesson-tutor
+    position: relative
+
+    margin: 5px 0 5px 0
+
+.avatar
+    display: inline-block
+
+    width: 20.62px
+    height: 20.62px
```
Comment (Anatolii-Ivanov) [inline frontend/src/app/modules/lessons/lessons-page/lessons-page.component.sass:111]: No decimals in css please

File: frontend/src/app/shared/components/youtube-player/youtube-player.component.sass [hunk 4]
```diff
@@ -0,0 +1,2 @@
+.container
+    background-color: #000
```
Comment (Anatolii-Ivanov) [inline frontend/src/app/shared/components/youtube-player/youtube-player.component.sass:2]: Use variables for colors please

File: frontend/src/app/shared/components/youtube-player/youtube-player.component.ts [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+@Component({
+  selector: 'app-youtube-player',
+  templateUrl: './youtube-player.component.html',
+  styleUrls: ['./youtube-player.component.sass']
+})
+export class YoutubePlayerComponent implements OnInit {
+
+    apiLoaded = false;
+
+    videoId = 'xqAriI87lFU';
```
Comment (Anatolii-Ivanov) [inline frontend/src/app/shared/components/youtube-player/youtube-player.component.ts:12]: Make it as Input

File: frontend/src/app/shared/components/youtube-player/youtube-player.component.ts [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+    ngOnInit() {
+        this.videoWidth = window.innerWidth * 0.8;
+        this.videoHeight = window.innerHeight * 0.8 - 4;
+
+        if (!this.apiLoaded) {
+        const tag = document.createElement('script');
+        tag.src = 'https://www.youtube.com/iframe_api';
+        document.body.appendChild(tag);
+        this.apiLoaded = true;
```
Comment (Anatolii-Ivanov) [inline frontend/src/app/shared/components/youtube-player/youtube-player.component.ts:24]: Formatting
Are you sure this is needed?

File: frontend/src/app/shared/shared.module.ts [hunk 7]
```diff
@@ -3,12 +3,16 @@ import { NgModule } from '@angular/core';
...
+
 import { LoadingSpinnerComponent } from './components/loading-spinner/loading-spinner.component';
 import { NotFoundComponent } from './components/not-found/not-found.component';
+import { YoutubePlayerComponent } from './components/youtube-player/youtube-player.component';
 
 @NgModule({
-    imports: [CommonModule, RouterModule, FormsModule, ReactiveFormsModule, RouterModule],
-    declarations: [LoadingSpinnerComponent, NotFoundComponent],
+    imports: [CommonModule, RouterModule, FormsModule, ReactiveFormsModule, RouterModule, MatDialogModule, YouTubePlayerModule,],
+    declarations: [LoadingSpinnerComponent, NotFoundComponent, YoutubePlayerComponent],
```
Comment (Anatolii-Ivanov) [inline frontend/src/app/shared/shared.module.ts:15]: New import - new line

File: frontend/src/app/modules/lessons/lessons-page/lessons-page.component.html [hunk 8]
```diff
@@ -0,0 +1,508 @@
+<div class="container">
+    <div class="date-container">
+        <h2 class="date">05 March 2023, Tuesday</h2>
+    </div>
+
+    <div class="lessons-section">
+        <div class="card">
```
Comment (Anatolii-Ivanov) [inline frontend/src/app/modules/lessons/lessons-page/lessons-page.component.html:7]: ngFor please

File: frontend/src/app/modules/lessons/lessons-page/lessons-page.component.html [hunk 9]
```diff
@@ -0,0 +1,508 @@
...
+            <div class="photo" (click)="openDialog()">
+                <img src="../../../../assets/lesson-mocks/Photo-4.png"
```
Comment (Anatolii-Ivanov) [inline frontend/src/app/modules/lessons/lessons-page/lessons-page.component.html:9]: Make it loaded from ts

File: frontend/src/app/modules/lessons/lessons-page/lessons-page.component.html [hunk 10]
```diff
@@ -0,0 +1,508 @@
...
+                    alt="thumbnail">
+                <div class="circle">
+                    <svg width="36" height="36" viewBox="0 0 36 36" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <circle opacity="0.5" cx="18" cy="18" r="18" fill="white" />
+                    </svg>
```
Comment (Anatolii-Ivanov) [inline frontend/src/app/modules/lessons/lessons-page/lessons-page.component.html:14]: Svgs to separate files please

Comment (Anatolii-Ivanov) [reply frontend/src/app/modules/lessons/lessons-page/lessons-page.component.html:14]: Everywhere

File: frontend/src/app/modules/lessons/lessons-page/lessons-page.component.html [hunk 11]
```diff
@@ -0,0 +1,508 @@
...
+                <div class="play">
+                    <svg width="11" height="13" viewBox="0 0 11 13" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <path d="M1.125 0.875L9.875 6.5L1.125 12.125V0.875Z" fill="#0B080A" stroke="#0B080A"
+                            stroke-width="1.41667" stroke-linecap="round" stroke-linejoin="round" />
+                    </svg>
+                </div>
+            </div>
+
+            <div class="info">
+                <h3 class="lesson-theme">The Real New Yorker’s Sandwich</h3>
```
Comment (Anatolii-Ivanov) [inline frontend/src/app/modules/lessons/lessons-page/lessons-page.component.html:25]: Data from ts

File: frontend/src/app/modules/lessons/lessons-page/lessons-page.component.html [hunk 12]
```diff
@@ -0,0 +1,508 @@
...
+                            fill="#ED4C5C" />
+                    </svg>
+                    <p class="tutor-name">Roger Vaccaro</p>
+                </div>
+                <div class="lesson-topics">
+                    <div class="topic">healthy</div>
+                    <div class="topic">food</div>
+                    <div class="topic">diet</div>
+                    <div class="topic">biology</div>
```
Comment (Anatolii-Ivanov) [inline frontend/src/app/modules/lessons/lessons-page/lessons-page.component.html:42]: Data from ts file

**Review by Anatolii-Ivanov — CHANGES_REQUESTED** [review]

**Comment by tatianahutii** [comment]
@ayomanetsan Could you please add screenshots?

File: frontend/src/app/modules/lessons/lessons-page/lessons-page.component.html [hunk 13]
```diff
@@ -0,0 +1,53 @@
...
+                <img src="../../../../assets/lesson-icons/rocket-icon.svg" alt="rocket-icon">
+                <span class="count">{{ lesson.level }}</span>
+            </div>
+
+            <div class="lesson-actions">
+                <button class="join" [disabled]="lesson.isDisabled" [textContent]="lesson.isDisabled ? 'Expired' : 'Join'"></button>
+                <a class="expand" href="">See questions</a>
+                <svg width="11" height="7" viewBox="0 0 11 7" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <path d="M1.25 1.375L5.5 5.625L9.75 1.375" stroke="#1C2FBA" stroke-width="1.41667"
+                        stroke-linecap="round" stroke-linejoin="round" />
```
Comment (Anatolii-Ivanov) [inline frontend/src/app/modules/lessons/lessons-page/lessons-page.component.html:48]: svg

**Review by Anatolii-Ivanov — CHANGES_REQUESTED** [review]

**Review by Anatolii-Ivanov — APPROVED** [review]

**Review by SidletskyiOlexandr — APPROVED** [review]

**Pull request merged**